kernel/kernbench.py

```diff
@@ -118,7 +118,7 @@ def test(self):
         """
         # Setting the kernel
         # Uncompress the kernel archive to the work directory
-        tarball = self.fetch_asset("kernbench.zip", locations=self.location,
+        tarball = self.fetch_asset("kernbench.zip", locations=[self.location],
```

Comment:
@harish-24 looks good, but would extending the expiry also be good ?

Reply:
It can be increased, but the idea was to run daily so that latest kernel would be fetched everyday.

Reply:
Oh ok